app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,23 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find params[:followed_id]
+
+    current_user.follow(@user)
+    respond_to do |format|
+      format.html{redirect_to @user}
+      format.js
+    end
+  end
+
+  def destroy
+    @user = Relationship.find(params[:id]).followed
+
+    current_user.unfollow(@user)
```

Comment:
```suggestion
    current_user.unfollow @user
```
có trường hợp ko tìm thấy @user ko nhỉ?